library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapter.java

```diff
@@ -51,7 +56,7 @@ public V instantiateItem(ViewGroup container, int position) {
      * @param position  the position of the data set that is to be represented by this view
      * @param viewState the state of the view
      */
-    protected void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
+    private void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
```

Comment:
This method always calls the overload (which is the one doing the actual binding) so you already have that capability. This should be private, I absolutely agree with Joe.